lib/modules/dosomething/dosomething_api/resources/kudos_resource.inc

```diff
@@ -189,6 +188,29 @@ function _kudos_resource_index($reportbackitems, $count) {
  * @param $term_ids
  */
 function _kudos_resource_create($reportback_item_id, $term_ids, $northstar_id) {
+  if (variable_get('rogue_collection', FALSE)) {
+    global $user;
+    $northstar_user = dosomething_northstar_get_user($user->uid, 'drupal_id');
```

Comment:
This still doesn't let us specify an arbitrary user ID. Ideally we want to use the current "session" when this API is hit via the web on legacy campaigns, but allow the `ds-api@dosomething.org` admin user to specify a custom ID (via a string passed to that `$northstar_id` parameter)  for other internal API requests.

Let's talk tomorrow and we can figure it out! 😄